packages/components/data-resource/s3-endpoint-type.tsx

```diff
@@ -30,14 +48,91 @@ type S3EndpointTypeProps = {
   namespace: string;
 };
 
+const isValidNS = (projName: string) => {
+  const isValid = EXCLUDED_PREFIX.reduce((acc, cur) => {
+    return acc && !projName.startsWith(cur);
+  }, true);
+  return isValid && !EXCLUDED_NS.includes(projName);
+};
+
+const { namespace, ...secretResourceWithoutNamespace } = secretResource;
```

Comment:
are we using this "namespace" anywhere ?
if no, no need to initialize unused variable...